src/linux/sentry_procmaps_modulefinder.c

```diff
@@ -275,7 +308,8 @@ load_modules(sentry_value_t modules)
         } else if (!module.file.len
```

Comment:
Should we be defensive here and bail if `module.start == 0`? I'm assuming it's rather unrealistic, but if `/proc/maps` returns a mapping at `0x0` and `linux_vdso` is null, you'll segfault later on.